bug: tsc error in Toast.stories.tsx

# ISSUE ✅ : tsc error in Toast.stories.tsx

## 📖 Summary

tsc 시 Toast.stories.tsx Description에서 에러 터짐
=> children deprecated 된 걸로 보임

## Error Script

src/components/Toast/Toast.stories.tsx:38:12 - error TS2559: Type '{ children: string; }' has no properties in common with type 'IntrinsicAttributes & DescriptionProps'.

38           <Description>Toast 컴포넌트를 사용하기 위한 Custom Hook입니다.</Description>
              ~~~~~~~~~~~

Found 1 error in src/components/Toast/Toast.stories.tsx:38

 ELIFECYCLE  Command failed with exit code 2.

## Photo

<img width="814" alt="image" src="https://github.com/yourssu/YDS-React/assets/84809236/22017540-b1ab-4948-9e2c-c7bfc8b89511">

--- a/src/components/Toast/Toast.stories.tsx
+++ b/src/components/Toast/Toast.stories.tsx
@@ -28,7 +28,7 @@ const meta: Meta<typeof Toast> = {
           </ol>
           <br />
           <Title>useToast</Title>
-          <span>'Toast 컴포넌트를 사용하기 위한 Custom Hook입니다.'</span>
+          <span>Toast 컴포넌트를 사용하기 위한 Custom Hook입니다.</span>
           <Markdown>{HookSource}</Markdown>
           <br />
           <Stories />

--- a/src/components/Toast/Toast.stories.tsx
+++ b/src/components/Toast/Toast.stories.tsx
@@ -1,11 +1,4 @@
-import {
-  Stories,
-  Primary as PrimaryBlock,
-  Controls,
-  Title,
-  Description,
-  Markdown,
-} from '@storybook/blocks';
+import { Stories, Primary as PrimaryBlock, Controls, Title, Markdown } from '@storybook/blocks';
 import { Meta, StoryObj } from '@storybook/react';
 
 import { useToast } from '@/hooks/useToast';
@@ -35,7 +28,7 @@ const meta: Meta<typeof Toast> = {
           </ol>
           <br />
           <Title>useToast</Title>
-          <Description>Toast 컴포넌트를 사용하기 위한 Custom Hook입니다.</Description>
+          <span>'Toast 컴포넌트를 사용하기 위한 Custom Hook입니다.'</span>
           <Markdown>{HookSource}</Markdown>
           <br />
           <Stories />

